Provide cycle and leadtime for baseline.url rendering (#50)
maddenp-cu/wxvx: README.md

### baseline.url

> The `baseline.url` value may include Jinja2 expressions, processed at run-time with [`jinja2.Template.render()`](https://jinja.palletsprojects.com/en/stable/api/#jinja2.Template.render). Variables `yyyymmdd` (cycle date, a `str`), `hh` (cycle time, a `str`), and `fh` (forecast hour, aka leadtime, an `int`) will be supplied by `wxvx`. Additionally, `cycle` (a `datetime`) and `leadtime` (a `timedelta`) are available. Use `cycle.strftime()` to format date/time components (e.g., `'%Y%m%d'`, `'%H'`), and compute valid time with `(cycle + leadtime)`.

Reviewer: ```suggestion
The `baseline.url` value may include Jinja2 expressions, processed at run-time with [`jinja2.Template.render()`](https://jinja.palletsprojects.com/en/stable/api/#jinja2.Template.render). Variables `yyyymmdd` (cycle date, a `str`), `hh` (cycle time, a `str`), and `fh` (forecast hour, aka leadtime, an `int`) will be supplied by `wxvx`. Additionally, `cycle` (a [`datetime`](https://docs.python.org/3/library/datetime.html#datetime-objects)) and `leadtime` (a [`timedelta`](https://docs.python.org/3/library/datetime.html#timedelta-objects)) are available for computing arbitrary time strings.
```
I'm torn on this, but maybe we should say less here: `wxvx` itself computes all the in-play validtimes based on the product of `cycles` and `leadtimes`, and uses them to render potentially different `baseline.url` and `forecast.path` values each time it needs one, so I think mentioning valid time here might confuse users, who probably (?) shouldn't need to compute this themselves. Would it be ok to leave it somewhat vague, with links to the Python docs for people to consult when deciding how they want to use these objects?

All this applies to the `forecast.path` docs, below, too. If it seems too long to repeat, please feel free to factor it out to another section and link to that from these sections.

or

``` yaml

> leadtimes: [3, 6, 9]

Reviewer: Oof, thanks.
